composer.json

```diff
@@ -49,6 +49,7 @@
         "openeuropa/oe_paragraphs": "dev-master",
         "openeuropa/oe_search": "~1.0",
         "openeuropa/oe_webtools": "dev-master",
+        "openeuropa/oe_contact_forms": "dev-OPENEUROPA-3282",
```

Comment:
This should be put on master after the https://github.com/openeuropa/oe_contact_forms/pull/12 is merged.

Reply:
I think after we ensure we are more or less done there, we will be able to release and use a tag as wel.

Reply:
fixes where needed so I've put a different branch for now